helm/hpcc/templates/_helpers.tpl

```diff
@@ -149,6 +293,10 @@ Add the secret volume mounts for a component
 {{- end -}}
 {{- end -}}
 
+{{- define "hpcc.echo" -}}
```

Comment:
Is this used? A debugging aid?

If it's used it should have a comment describing what it is. If it's a debugging aid, perhaps it can now be removed?

Reply:
should be deleted.  It was an experiment